usr/local/www/interfaces_groups_edit.php

```diff
@@ -168,193 +172,76 @@
-    <td width="78%" class="vtable">
-      <table id="maintable" summary="main table">
-        <tbody>
-          <tr>
-            <td><div id="onecolumn"><?=gettext("Interface");?></div></td>
-          </tr>
-
-	<?php
-	$counter = 0;
-	$members = $pconfig['members'];
-	if ($members <> "") {
-		$item = explode(" ", $members);
-		foreach($item as $ww) {
-			$members = $item[$counter];
-			$tracker = $counter;
-	?>
-        <tr>
-	<td class="vtable">
-	        <select name="members<?php echo $tracker; ?>" class="formselect" id="members<?php echo $tracker; ?>">
-			<?php
-				$found = false;
-				foreach ($iflist as $ifnam => $ifdescr) {
-					echo "<option value=\"{$ifnam}\"";
-					if ($ifnam == $members) {
-						$found = true;
-						echo " selected=\"selected\"";
-					}
-					echo ">{$ifdescr}</option>";
-				}
 
-				if ($found === false)
-					foreach ($iflist_disabled as $ifnam => $ifdescr)
-						if ($ifnam == $members)
-							echo "<option value=\"{$ifnam}\" selected=\"selected\">{$ifdescr}</option>";
-			?>
-                        </select>
-	</td>
-        <td>
-	<a onclick="removeRow(this); return false;" href="#"><img border="0" src="/themes/<?echo $g['theme'];?>/images/icons/icon_x.gif" alt="remove" /></a>
-	      </td>
-          </tr>
-<?php
-		$counter++;
+if (isset($id) && $a_ifgroups[$id]) {
+	$section->addInput(new Form_Input(
+		'id',
+		'hidden',
+		htmlspecialchars($id)
+	));
+}
 
-		} // end foreach
-	} // end if
-?>
-        </tbody>
-		  </table>
-			<a onclick="javascript:addRowTo('maintable'); return false;" href="#">
-        <img border="0" src="/themes/<?= $g['theme']; ?>/images/icons/icon_plus.gif" alt="" title="<?=gettext("add another entry");?>" />
-      </a>
-		<br /><br />
-		<strong><?PHP echo gettext("NOTE:");?></strong>
-		<?PHP echo gettext("Rules for WAN type interfaces in groups do not contain the reply-to mechanism upon which Multi-WAN typically relies.");?>
-		<a href="https://doc.pfsense.org/index.php/Interface_Groups"><?PHP echo gettext("More Information");?></a>
-		</td>
-  </tr>
-  <tr>
-    <td width="22%" valign="top">&nbsp;</td>
-    <td width="78%">
-      <input id="submit" name="submit" type="submit" class="formbtn" value="<?=gettext("Save");?>" />
-      <a href="interfaces_groups.php"><input id="cancelbutton" name="cancelbutton" type="button" class="formbtn" value="<?=gettext("Cancel");?>" /></a>
-      <?php if (isset($id) && $a_ifgroups[$id]): ?>
-      <input name="id" type="hidden" value="<?=htmlspecialchars($id);?>" />
-      <?php endif; ?>
-    </td>
-  </tr>
-</table>
-</form>
-
-<script type="text/javascript">
-//<![CDATA[
-	field_counter_js = 1;
-	rows = 1;
-	totalrows = <?php echo $counter; ?>;
-	loaded = <?php echo $counter; ?>;
-//]]>
-</script>
+$form->setSubmit(new Form_Input(
+	'Submit',
+	'submit',
+	gettext('Save')
+))->forceName('Submit')->removeClass('form-control')->addClass('btn btn-primary');
```

Comment:
Please remove the add&remove classes here. If we want all submit-buttons to have those I'll add them to setSubmit